### DISCUSSION TITLE
Windows 11 /10 Treiber

### DISCUSSION THREAD

**Discussion by danzbox** [R3NE07/Futro-S740#44] [Q&A]
Hat jemand mal versucht Windows 10/11 zu installieren und kann was zu den notwendigen Treibern sagen. Auf den support Seiten von Fujitsu gibt es ja nur noch ältere Windows 10 Treiber und nach der Erstinstallation habe ich super viele unbekannte Geräte im Gerätemanager. Windows lädt auch nicht gerade viele Treiber online nach. Ich habe ja nix gegen etwas ältere Treiber möchte aber auch keine unnötige Bloatware installieren...

**Comment by Hummtaro** [comment]
Da wir hier einen gängigen Intel Chipsatz haben, sollte es mit einem Intel Chipsatz Treiber fast getan sein.
Ich glaube ich habe bei mir einfach den Snappy Driver Installer alles fehlende installieren lassen
